the-collab-lab/tcl-27-smart-shopping-list
# Move app.js to src and consolidate components folder

## Description

Updated dependencies and consolidated code from duplicate app.js files into one app.js file and moved it to the src folder. Also consolidated duplicate components/containers folders.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓ | :bug: Bug fix              |
|    | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
| ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

* From your terminal, pull down this branch with git pull origin sk-dp-move-app-to-src and check that branch out with git checkout sk-dp-move-app-to-src
* Then npm/yarn to install the newly added dependencies locally and npm/yarn start to launch the app.
* Fill out the form with a new item. Make sure it adds it to the list and make sure that item appears in the items collection of the database here. Also check to see if the bottom nav bar routes to the correct components.
